eclair-core/src/main/scala/fr/acinq/eclair/channel/Channel.scala

```diff
@@ -1816,6 +1796,9 @@ class Channel(val nodeParams: NodeParams, val wallet: EclairWallet, remoteNodeId
 
     case Event(c: CMD_UPDATE_RELAY_FEE, d) => handleCommandError(CommandUnavailableInThisState(d.channelId, "updaterelayfee", stateName), c)
 
+    // at restore, if the configuration has changed, the channel will send a command to itself to update the relay fees
+    case Event(RES_SUCCESS(_: CMD_UPDATE_RELAY_FEE, channelId), d: DATA_NORMAL) if channelId == d.channelId => stay()
```

Comment:
Can't we set `replyTo` to the deadletters actor instead of `ActorRef.noSender` to avoid that?

Reply:
We could, but we would get `WARN` logs from the `Deadletters` handlers. I think it's better to handle them explicitly, since we know they will happen, and leave `Deadletters` for unexpected messages.

Reply:
I was hoping it wouldn't log anything in that case, but then let's keep it like that!